### ISSUE TITLE
Update for last version error

### ISSUE THREAD

**Issue by robertbigk** [tuya/tuya-home-assistant#209]
Hi i update for last version and integration stop error all time
![msedge_6lMj8B0PCH](https://user-images.githubusercontent.com/34956477/126879417-9a6d9bd2-56e7-4ca2-914a-8000bf6ff967.png)
[home-assistant.log](https://github.com/tuya/tuya-home-assistant/files/6872989/home-assistant.log)

**Comment by danmed** [comment]
I'm seeing the same thing on my end.

**Comment by Martynhsmith** [comment]
I went from an earlier version to 1.4 (1.2 I think).
I was getting the same error 
I deleted the integration and reset it up and it now works

**Comment by tsutsuku** [comment]
Try v1.4.1.

**Comment by robertbigk** [comment]
Still no work sorry
![msedge_6lMj8B0PCH](https://user-images.githubusercontent.com/34956477/126925900-92e7bddb-616b-4060-9db2-14c3deeaccf5.png)

**Comment by robertbigk** [comment]
[home-assistant.log](https://github.com/tuya/tuya-home-assistant/files/6875217/home-assistant.log)

**Comment by t0ny-peng** [comment]
I can confirm that removing the old integration and adding a new one with the fix in v1.4.1 works for me.

**Comment by robertbigk** [comment]
Yes but I got 40 item all described if I deleted i need to sort all again that I try to don't do it:)

**Comment by t0ny-peng** [comment]
@robertbigk I see. I didn't change the eneity id of the entities added by Tuyav2, so all my lovelace interface and automations work as before. 

I would suggest you also keep the eneity_id untouched since Tuyav2 is still under heavy development and is likely to break one in a while

**Comment by robertbigk** [comment]
![msedge_S31APYMjIH](https://user-images.githubusercontent.com/34956477/126935089-7d39c2d1-74e2-44b2-8646-7fd7dfcc3773.png)
Now I got all time password is not correct I got this bug but I don't know how I resolve then. 

I copy paste than no possible to wrong data

id from tuya
secret from tuya
account smatlife
pass smartlife

**Comment by robertbigk** [comment]
![msedge_UlnUR72tTd](https://user-images.githubusercontent.com/34956477/126935557-5ebbf2f1-f473-40dc-9183-15f286d257a6.png)
Again water for cast bw-spf1 uv not work

**Comment by tsutsuku** [comment]
@robertbigk  I think you select the wrong project type.

**Comment by tsutsuku** [comment]
> Still no work sorry
> ![msedge_6lMj8B0PCH](https://user-images.githubusercontent.com/34956477/126925900-92e7bddb-616b-4060-9db2-14c3deeaccf5.png)

Something wrong with your code, the v1.4.1 should like this:
<img width="748" alt="WeChatWorkScreenshot_ca6a4033-c3fb-4767-9519-6e0ab869c4a8" src="https://user-images.githubusercontent.com/18442047/126942865-ced087e6-7887-466c-a6a7-6c514dc35c83.png">

**Comment by robertbigk** [comment]
Project type is fine i reinstall all delete tuya installation and install again and bw-spf1 uv still not work also on 1.41
